package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "systelab-components-wdio-test",
-  "version": "1.9.0",
+  "version": "2.0.0",
```

Comment:
Considering this is not a breaking changes (just a new feature), new version number should be 1.10.0